src/Integration.Vsix.UnitTests/SonarAnalyzer/SonarAnalyzerManagerForBoundTests.cs

```diff
@@ -18,12 +18,12 @@
 namespace SonarLint.VisualStudio.Integration.UnitTests.SonarAnalyzer
 {
     [TestClass]
-    public class SonarAnalyzerDeactivationManagerForBoundTests
+    public class SonarAnalyzerManagerForBoundTests
```

Comment:
There's already a class named `SonarAnalyzerManagerTests`, there are many tests for this class, and tried to separate them into tow logic group in two classes.